nativelib/src/main/scala/scala/scalanative/posix/grp.scala

```diff
@@ -0,0 +1,10 @@
+package scala.scalanative
+package posix
+
+import native.extern
+
+/**
+ * Created by remi on 28/02/17.
+ */
```

Comment:
Please remove all authorship comments from the code. We don't use them as they easily get out of date.